Assets/Scripts/Diablerie/Game/UI/Menu/ClassSelect/ClassSelector.cs

```diff
@@ -0,0 +1,277 @@
+                        Fps = 15
+                    }
+                },
+                {
+                    ClassSelectorState.FrontIdle,
+                    new ClassSelectorStateInfo
+                    {
+                        Sprites = frontIdleSprites.GetSprites(0),
+                        OverlaySprites = classAnimationInfo.HasFrontIdleOverlay ? frontIdleOverlaySprites?.GetSprites(0) : null,
+                        Loop = true,
+                        HideOnFinish = false,
+                        SortingOrderShift = 0,
+                        Material = classAnimationInfo.OverlayMaterial,
+                        Fps = 25
+                    }
+                },
+                {
+                    ClassSelectorState.BackTransition,
+                    new ClassSelectorStateInfo
+                    {
+                        Sprites = backTransitionSprites.GetSprites(0),
+                        OverlaySprites = classAnimationInfo.HasBackTransitionOverlay ? backTransitionOverlaySprites?.GetSprites(0) : null,
+                        Loop = false,
+                        HideOnFinish = true,
+                        SortingOrderShift = -10,
+                        Material = classAnimationInfo.OverlayMaterial,
+                        SfxPath = backTransitionSfxPath,
+                        Fps = 25
+                    }
+                },
+                {
+                    ClassSelectorState.FrontTransition,
+                    new ClassSelectorStateInfo
+                    {
+                        Sprites = frontTransitionSprites.GetSprites(0),
+                        OverlaySprites = classAnimationInfo.HasFrontTransitionOverlay ? frontTransitionOverlaySprites?.GetSprites(0) : null,
+                        Loop = false,
+                        HideOnFinish = true,
+                        SortingOrderShift = 10,
+                        Material = classAnimationInfo.OverlayMaterial,
+                        SfxPath = frontTransitionSfxPath,
+                        Fps = 25
+                    }
+                }
+            };
+
+            var position = _rectTransform.position;
+            
+            _main = UiHelper.CreateAnimatedObject("main", loop: false);
+            _main.transform.position = UiHelper.ScreenToWorldPoint(position);
+            _main.transform.parent = _rectTransform;
+
+            _overlay = UiHelper.CreateAnimatedObject("overlay", loop: false, sortingOrder: classAnimationInfo.OverlaySortingOrder);
+            _overlay.transform.position = UiHelper.ScreenToWorldPoint(position);
+            _overlay.transform.parent = _rectTransform;
+
+            _mainAnimator = _main.GetComponent<SpriteAnimator>();
+            _mainAnimator.OnFinish += MainAnimatorOnFinish;
+            _mainAnimator.useUnscaledTime = true;
+            
+            _overlayAnimator = _overlay.GetComponent<SpriteAnimator>();
+            _overlayAnimator.useUnscaledTime = true;
```

Comment:
IT's not necessary, because timeScale in menu is always equal to 1.

Reply:
Removed